services/s3/src/main/java/software/amazon/awssdk/services/s3/presigner/S3Presigner.java

```diff
@@ -123,7 +123,25 @@
  * being browser-compatible.
  * <p />
  *
- * <b>Executing a Pre-Signed Request from Java code</b>
+ * <h3>Configurations that affect browser compatibility</h3>
+ * <h4>Enabling Checking Validation</h4>
+ * If checksum validations are enabled, the presigned URL will no longer be browser compatible because it adds a signed header
+ * that must be included in the HTTP request.
+ *
+ * Checksum validation is disabled in the presigner by default, but when using a custom {@link S3Configuration}, it must be
+ * explicitly disabled:
+ *
+ * <pre>
+ *         S3Presigner presigner = S3Presigner.builder()
+ *                                            .serviceConfiguration(S3Configuration.builder()
+ *                                                                                 .checksumValidationEnabled(false)
+ *                                                                                 .accelerateModeEnabled(true)
```

Comment:
Can we remove `accelerateModeEnabled`? It seems we are suggesting it needs to be enabled to be browser compatible

Reply:
I wanted to show a "real world" use case when a customer might want to specify a custom `S3Configuration` in the first place, but that's a good point. I removed it and clarified the `S3Configuration` use case in the docs.